server/routes/api.js

```diff
@@ -6,7 +6,22 @@ const moment = require('moment')
 const requestPromise = require("request-promise")
 const Favorites = require("../models/Favorites")
 
-// const KeyID = ""
+const APPID = "693487d5ce7f67db0872c3ce4dbe3b15"
+const WheatherAPIbasicURL = "https://api.openweathermap.org/data/2.5/weather"
 
+router.post('/cityWeather', async function (req, res) {
+    const cityName=req.body.cityName
+    const countryName=req.body.countryName
+    try {
+        const weatherData =  await requestPromise(`${WheatherAPIbasicURL}?q=${cityName},q=${countryName}&units=metric&APPID=${APPID}`)
+        res.send(JSON.parse(weatherData))
```

Comment:
Hey,
Can you please res,send an object only with the relevant info? also with the conversion of the icons urls already built in. This is something that should be done before the data arrives to the front-end.